packages/ember-toucan-form/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@crowdstrike/ember-toucan-form",
-  "version": "0.2.1",
+  "version": "1.0.0",
```

Comment:
This should be `0.3.0`.  For some reason changesets is bumping it to a major version